public/js/NeighbourImages.js

```diff
@@ -6,14 +6,26 @@ var _ = require('lodash');
  * @constructor
  */
 function NeighbourImages() {
-    var self = this;
-
     this.neighbours = [];
     this.selectedImage = "";
     this.neighbourImages = [];
 
-    var divWidth = Math.round($('#image-column').width()) - 20;
-    var imgWidth = Math.round(divWidth/3) - 20;
+    // cache jquery selectors used frequency by gallery methods
+    // call these in the constructor so the DOM is traversed only once
+    // before caching.
```

Comment:
There is the promise of greatness in this docstring! =) But it's still a bit off that: 

- is this just aliasing the names? Or what?
- what's the alternative design? (not in the constructor) I don't understand why this is more efficient than calling them on demand.